src/all_enum_values.h

```diff
@@ -14,7 +14,7 @@ template<typename E, size_t... I>
 auto all_enum_values_helper( std::index_sequence<I...> ) ->
 const std::array<E, num_enum_values<E>()> &
 {
-    static constexpr std::array<E, num_enum_values<E>()> result{ static_cast<E>( I )... };
+    static constexpr std::array<E, num_enum_values<E>()> result{ { static_cast<E>( I )... } };
```

Comment:
Alas. I was hoping we no longer had to care about versions of clang with this bug. I guess you can't upgrade for the usual MacOS reasons. But it's strange to me that you had trouble when the Mac CI didn't...

Reply:
For the context of anyone else following this discussion: it is just a warning.  The bug that got fixed relates to improving the heuristics about when the warning is reported.  So it's not a conformance issue, just a QoL issue.